src/components/ReviewList.jsx

```diff
@@ -22,29 +24,49 @@ function ReviewList() {
       })
       .then((res) => {
         setReviews(res.data.reviews);
+      })
+      .catch((err) => {
+        setErr("404 category not found");
+      });
```

Comment:
this isn't necessarily erroring because the category wasn't found!

It could error from the network being down or a number of other reasons!

Reply:
Changed to "Something went wrong"